gitlab.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "GitLab",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/pandorixtech/iocage-plugin-gitlab.git",
+  "pkgs": [
+    "www/gitlab-ce",
+    "postgresql11-server",
+    "postgresql11-contrib",
+    "postgresql11-client",
+    "nginx",
+    "gtar"
+  ],
+  "properties": {
+     "dhcp": "on",
+     "allow_sysvipc": "1"
+  },
+  "packagesite": "http://pkg.FreeBSD.org/${ABI}/latest",
+  "fingerprints": {
+	  "iocage-plugins": [
+		  {
+		  "function": "sha256",
+                  "fingerprint": "b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438"
+	  }
+	  ]
+  },
+  "official": true
```

Comment:
Shouldn't this be `"official": false` now that it has been moved to this community plugin index (and to match the INDEX official value)?

Reply:
Correct! Thank you for your review, I've just pushed a fix for it.